keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ResourceEvent.kt

```diff
@@ -442,3 +443,22 @@ data class ResourceCheckError(
     exception.message
   )
 }
+
+data class ResourceDiffNotActionable(
+  override val kind: ResourceKind,
+  override val id: String,
+  override val application: String,
+  override val timestamp: Instant,
+  override val message: String?
```

Comment:
It doesn't look like we're using this constructor anywhere. Could just make the one below the default.

Reply:
this is the data class definition, we need that

Reply:
oh i see what you're saying. I just want to leave it the same as the other events though